packages/@uppy/companion/src/server/controllers/s3.js

```diff
@@ -353,7 +357,62 @@ module.exports = function s3 (config) {
     }, next)
   }
 
+  const policy = {
+    Version: '2012-10-17',
```

Comment:
2012 version?

Reply:
Yes, that's the latest (it looks like they are as quick to provide API updates as they are for upgrading the CSS of amazon.com, but that kind of stability is actually very good for us in this case)

Reply:
```suggestion
    Version: '2012-10-17' // not an old version, this is latest,
```

Reply:
```suggestion
    Version: '2012-10-17', // latest at the time of writing
```